gallery/activitylog-insights-Azure Monitor.json

```diff
@@ -0,0 +1,19 @@
+{
+	"$schema": "https://raw.githubusercontent.com/microsoft/Application-Insights-Workbooks/master/schema/gallery.json",
```

Comment:
do we want to standardize on tolower of the filenames?  is it currently case specific?  (or does the /should the package generation be tolowering everything?  or will that break things?)

Reply:
Good question. I haven't actually tried, as I just used a script to generate all these files from the existing gallery content.